web/locale/zh_Hans/LC_MESSAGES/django.po

```diff
@@ -320,7 +322,7 @@ msgstr "关注"
 
 #: templates/auth/profile.html:32 templates/stats/profile.html:21
 msgid "KARMA"
-msgstr ""
+msgstr "节操"
```

Comment:
Waiting for my IR parser to dump a meaning for each of the words in the contexts I have learned so I can make a diff.